Ballerina.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 org-name = "wso2"
-version = "0.8.2"
+version = "0.8.4"
```

Comment:
Why aren't we upgrading the version to 0.8.3?

Reply:
latest version of this connector in BC is 0.8.3. That's why I put 0.8.4